src/data/forecastData.json

```diff
@@ -0,0 +1,1520 @@
+{
```

Comment:
Is the data in this file used anywhere? If not and it doesnt have any other use it should be deleted